app/templates/_bower.json

```diff
@@ -38,6 +38,7 @@
         "sockjs-client": "1.0.3",
         "stomp-websocket": "2.3.4",
         <%_ } _%>
+        "swagger-ui": "jhipster-needle-bower-add-dependency - JHipster will add additional bower dependency",
```

Comment:
I know, but it was the only way to add the needle without breaking the bower install.
- I try to add in ignore package, but still have an error
- I find an other solution, to add `"ignoredDependencies": ["jhipster-needle-bower-add-dependency"]` in `.bowerrc`, it's work when you execute the `bower install`, but t's not working with `grunt:wiredep` :(

The only solution it to use an existing dependency and put before.

Or create an specific package like `jhipster-needle` in bower?